packages/playground/src/main.ts

```diff
@@ -22,6 +23,19 @@ app.config.errorHandler = error => {
   );
 };
 
+Sentry.init({
```

Comment:
How does it behave if no DSN configured?

Reply:
no log errors in playground and no syncing with sentry